commcare_connect/accounting/migrations/0001_initial.py

```diff
@@ -0,0 +1,46 @@
+# Generated by Django 4.2.5 on 2024-09-12 16:16
+
+from decimal import Decimal
+from django.db import migrations, models
+
+
+def create_task(apps, schema_editor):
+    schedule, _ = CrontabSchedule.objects.get_or_create(
```

Comment:
I have followed the existing way of defining periodic tasks. But I am wondering why are we using the DatabaseScheduler instead of simply defining the schedules via the settings.CELERY_BEAT_SCHEDULE?